github/downloader_test.go

```diff
@@ -0,0 +1,205 @@
+		require.Equal(t, test.createdAt, repository.CreatedAt.String(), "DownloadRepository(%s, %s) CreatedAt %s not matching %s", test.owner, test.repository, test.createdAt, repository.CreatedAt.String())
+		require.Equal(t, test.isPrivate, repository.IsPrivate, "DownloadRepository(%s, %s) IsPrivate %s not matching %s", test.owner, test.repository, test.isPrivate, repository.IsPrivate)
+		require.Equal(t, test.isArchived, repository.IsArchived, "DownloadRepository(%s, %s) IsArchived %s not matching %s", test.owner, test.repository, test.isArchived, repository.IsArchived)
+		require.Equal(t, test.hasWiki, repository.HasWikiEnabled, "DownloadRepository(%s, %s) HasWikiEnabled %s not matching %s", test.owner, test.repository, test.hasWiki, repository.HasWikiEnabled)
+		require.Equal(t, test.numOfPrs, teststore.GetNumOfPrs(), "DownloadRepository(%s, %s) number of PRs %s not matching %s", test.owner, test.repository, test.numOfPrs, teststore.GetNumOfPrs())
+		require.Equal(t, test.numOfPrComments, teststore.GetNumOfPrComments(), "DownloadRepository(%s, %s) number of PR Comments %s not matching %s", test.owner, test.repository, test.numOfPrComments, teststore.GetNumOfPrComments())
+	}
+
+}
+
+func TestOnlineOrganizationDownload(t *testing.T) {
+	var tests = []struct {
+		org               string
+		version           int
+		url               string
+		createdAt         string
+		publicRepos       int
+		totalPrivateRepos int
+		numOfUsers        int
+	}{
+		{
+			"git-fixtures",
+			0,
+			"https://github.com/git-fixtures",
+			"2016-02-23 18:57:27 +0000 UTC",
+			8,
+			0,
+			0,
+		},
+	}
+
+	token := os.Getenv("GITHUB_TOKEN")
+
+	client := oauth2.NewClient(context.TODO(), oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: token},
+	))
+
+	var downloader, err = NewStdoutDownloader(client)
+
+	if err != nil {
+		t.Errorf("Downloader failed to init with error:%s", err)
+	}
+
+	saved := downloader.storer
+	defer func() { downloader.storer = saved }()
+	var teststore *store.TestStore = &store.TestStore{}
+	downloader.storer = teststore
+
+	for _, test := range tests {
+		err := downloader.DownloadOrganization(context.TODO(), test.org, test.version)
+		require.Nil(t, err, "DownloadOrganization(%s) failed", test.org)
+		org := teststore.GetOrganization()
+		require.Equal(t, test.org, org.Name, "Organization name should be the same")
+		require.Equal(t, test.url, org.Url, "Organization url should be the same")
+		require.Equal(t, test.createdAt, org.CreatedAt.String(), "Organization created date should be the same")
+		require.Equal(t, test.publicRepos, org.PublicRepos.TotalCount, "Organization public repos count should be the same")
+		require.Equal(t, test.totalPrivateRepos, org.TotalPrivateRepos.TotalCount, "Organization total private repos count should be the same")
+		require.Equal(t, test.numOfUsers, teststore.GetNumOfUsers(), "Organization total private repos count should be the same")
+	}
+}
```

Comment:
this tests structure is similar to the previous one so please take into account aforementioned comments